group_vars/all/globals.yml

```diff
@@ -36,3 +36,6 @@ postgresql_archive_command: >
   {% if wal_e_aws_instance_profile %}--aws-instance-profile{% endif %} wal-push %p
 
 postgres_master_host: "{{ deploy_env }}-postgres-master.{{ domain_name }}"
+
+elasticsearch_host_name: "{{ hosts_prefix }}-tsuru-elasticsearch"
+elasticsearch_url: "{{ deploy_env }}-elasticsearchapi.{{ domain_name }}"
```

Comment:
At least rename elasticsearch_url.
We can create a new story to standardise variable names.